src/lib/index/plannings.ts

```diff
@@ -36,8 +34,8 @@ const search = async (endpoint: URL, accessToken: string, params?: SearchPlannin
           {
             range: {
               'document.meta.core_planning_item.data.start_date': {
-                gte: start.toISOString(),
-                lte: end.toISOString()
+                gte: params?.where?.start,
```

Comment:
If we actually call this method without providing start or end options for the where clause, which is ok according to the interface, this search should break, won't it?.

Start and end should either be required or the range part should be added only when they are supplied.